src/doc/hmm.dox

```diff
@@ -125,7 +130,8 @@ context dependent HMM state depend on the following four things (you could view
 them as a 4-tuple):
```

Comment:
It would be clearer to describe this as 5-tuple... would lead to less potential confusion with the 4-tuple describing the transition-state.